modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -202,15 +211,21 @@ spec:
         address:
```

Comment:
Making a note for posterity.  In this example IPv6 DHCP is disabled, but we still set the IPv6 masquerade.  I don't think there will be an IPv6 default route, so we don't need to set the IPv6 `auto-route-metric`.     But if we did have a dual-stack config we would have to set the `auto-route-metric` for both IP families.

Reply:
As I mentioned above, we should be using dual-stack examples.  So let's also do that here.

Reply:
:heavy_check_mark: